sway/commands/output/hdr.c

```diff
@@ -0,0 +1,37 @@
+#include <strings.h>
+#include "sway/commands.h"
+#include "sway/config.h"
+#include "sway/output.h"
+#include "util.h"
+
+struct cmd_results *output_cmd_hdr(int argc, char **argv) {
+	if (!config->handler_context.output_config) {
+		return cmd_results_new(CMD_FAILURE, "Missing output config");
+	}
+	if (argc == 0) {
+		return cmd_results_new(CMD_INVALID, "Missing hdr argument");
+	}
+
+	bool current;
```

Comment:
```suggestion
    bool current = false;
```
Otherwise it'll fail to build because of Werror=maybe-uninitialized enabled

Reply:
Done